core/store/src/test/java/uk/gov/gchq/gaffer/store/operation/OperationChainValidatorTest.java

```diff
@@ -54,7 +54,7 @@ public void shouldValidateValidOperationChain() {
                 new GetElements(),
                 new ToVertices(),
                 new GetAdjacentIds()
-        )), true);
```

Comment:
Could you add in a test which results in a valid OperationChain being successfully validated?